{{cookiecutter.project_slug}}/Makefile

```diff
@@ -46,18 +46,22 @@ PRESET?=gpu-small
 HTTP_AUTH?=--http-auth
 
 # Command to run training inside the environment. Example:
-TRAIN_CMD=python -u $(CODE_DIR)/train.py --data $(DATA_DIR)
+TRAIN_CMD?=python -u $(CODE_DIR)/train.py --data $(DATA_DIR)
```

Comment:
Important bug fix: without this, `export TRAIN_CMD` (as stated in Readme) does not work